android/filament-android/src/main/java/com/google/android/filament/Scene.java

```diff
@@ -18,6 +18,26 @@
 
 import android.support.annotation.Nullable;
 
+/**
+ * A <code>Scene</code> is a flat container of {@link RenderableManager} and {@link LightManager}
+ * components.
+ * <br>
+ * <p>A <code>Scene</code> doesn't provide a hierarchy of objects, i.e.: it's not a scene-graph.
+ * However, it manages the list of objects to render and the list of lights, these can
+ * be added or removed from a <code>Scene</code> at any time.</p>
+ * <br>
+ * <p>A {@link RenderableManager} component <b>must</b> be added to a <code>Scene</code> in order
+ * to be rendered, and the <code>Scene</code> must be provided to a {@link View}.</p>
+ *
+ * <h1>Creation and Destruction</h1>
+ *
+ * A <code>Scene</code> is created using {@link Engine#createScene()} and destroyed using
+ * {@link Engine#destroyScene(Scene)}.
```

Comment:
There are several places in this file where you do `{@link Object#foo(..)}` but I had thought the javadoc syntax was  `{@link Object#foo}` (no args)

Reply:
It's the IDE's auto-complete that did this, I assumed it was the right way.